makefile

```diff
@@ -0,0 +1,41 @@
+BUILD_WEBAPP_DIR = ./webapp
+
+check-style:
+	@echo Checking style
+
+	gofmt -w .
+
+	cd $(BUILD_WEBAPP_DIR) && $(MAKE) check-style
+
+build:
```

Comment:
I feel like this should be moved to the top. That way, people can type in `make` and it'll default to building and not checking the style.